cmd/notation/registry.go

```diff
@@ -156,6 +157,9 @@ func getAuthClient(ctx context.Context, opts *SecureFlagOpts, ref registry.Refer
 
 	// build authClient
 	authClient := &auth.Client{
+		Client: &http.Client{
+			Timeout: 20 * time.Second,
```

Comment:
LGTM. But just curious, is `20 seconds` a standard for an auth client?

Reply:
`20s` is not just for auth but for the entire request (oauth2 flow + resources access). In a bad network scenarios, it can take several minutes.

Reply:
Yeah, seems we need more discussions on this.